app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  validates :name, presence: true
+
+  has_attached_file :avatar, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
```

Comment:
[No. right now it's storing in default directory. we can make it use S3 easily. ](https://github.com/thoughtbot/paperclip#understanding-storage)